db/sql.h

```diff
@@ -752,6 +748,15 @@ struct query_path_component {
     LINKC_T(struct query_path_component) lnk;
 };
 
+struct temptable {
+    int rootpage;
+    struct temp_cursor *cursor;
+    struct temp_table *tbl;
```

Comment:
IMO, `struct temptable` vs `struct temp_table` is a bit confusing.

Reply:
Agreed. I can cleanup in subsequent PR (struct wasn't introduced in the PR, that is what it was called already).

Reply:
I remember when both the structs were called `struct temp_table`. One day I couldn't take it anymore and renamed the one in `sqlglue.c`!

Reply:
https://news.ycombinator.com/item?id=14598201

Reply:
Indeed!